Makefile

```diff
@@ -199,15 +199,15 @@ else
 	@echo Creating namespace $(KAFKA_NAMESPACE)
 	@kubectl create namespace $(KAFKA_NAMESPACE) 2>&1 | grep -v "already exists" || true
```

Comment:
Shouldn't creating the namespace be outside  the if/endif as well then? as the namespace is need for the kafka instance.